falcon/media/base.py

```diff
@@ -0,0 +1,38 @@
+import abc
+
+import six
+
+
+@six.add_metaclass(abc.ABCMeta)
+class BaseHandler(object):
+    """Abstract Base Class for an internet media type handler"""
+
+    @abc.abstractmethod
+    def load(self):
```

Comment:
Good point... It made sense when we were using straight classes, but if we're creating instances now, then we can get rid of ``load()``